src/classes/DragNDropController.cls

```diff
@@ -0,0 +1,75 @@
+public with sharing class DragNDropController {
```

Comment:
Please add class header description. Should have the following format:

![image](https://user-images.githubusercontent.com/1637981/49878390-38bd7780-fe06-11e8-9f78-ec94d2f2bfd5.png)